### PR TITLE
Implement privacy risk score #78

### LINKED ISSUE #78: Create membership inference score, just as it is in Tensorflow privacy
See https://arxiv.org/pdf/2003.10595.pdf

Acceptance criteria:
- Implements the score as described in the paper
- Allows slicing according to the classes
- Allows slicing according to the correctly or wrongly classified data-points

### PR DESCRIPTION
**Pull request by blauertee**
- privacy risk score function utilizing the tf implementation
- test function for privacy risk score
- @jtorhoff s setup.py is included here as well
- added data/ to .gitignore
- added lossfn to Classifier class
- changed other tests to be compatible with lossfn as classifier arg.

should resolve #78

### CODE REVIEW

**Comment by blauertee** [comment]
Oh forgot to change the docs ... will do that :)

**Review by Erdnaf — APPROVED** [review]
looks very good

**Review by jtorhoff — APPROVED** [review]

File: privacy_evaluator/metrics/privacy_risk_score.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    :param attack_input: input data for compute membership probability
+    :param num_bins: the number of bins used to compute the training/test histogram
+    :return: membership probability results
+    """
+    loss_train = target.art_classifier.compute_loss(x_train, y_train)
+    loss_test = target.art_classifier.compute_loss(x_test, y_test)
+    return _compute_membership_probability(loss_train, loss_test, num_bins)
+
+
+def _compute_membership_probability(loss_train, loss_test, num_bins: int = 15):
```
Comment (marisanest) [inline privacy_evaluator/metrics/privacy_risk_score.py:34]: Can you please add the return types here (e.g. `_compute_membership_probability(loss_train, loss_test, num_bins: int = 15) -> Tupel[np.ndarray, np.ndarray]:`)

File: privacy_evaluator/metrics/privacy_risk_score.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+from privacy_evaluator.classifiers.classifier import Classifier
+import numpy as np
+
+
+def compute_privacy_risk_score(
```
Comment (marisanest) [inline privacy_evaluator/metrics/privacy_risk_score.py:5]: Same as below: Can you please add the return types here (e.g. `compute_privacy_risk_score(...) -> Tupel[np.ndarray, np.ndarray]:`)

**Review by marisanest — CHANGES_REQUESTED** [review]

**Review by marisanest — APPROVED** [review]
Great, thanks!

**Pull request merged**